packages/component-library/draft/Kaizen/Modal/Modal.elm

```diff
@@ -297,6 +298,19 @@ viewContent (Config config) =
                             Nothing ->
                                 confirmationConfig
 
+                    withOnConfirmDisabled confirmationConfig =
+                        case configs.onConfirm of
+                            Just confirmMsg ->
+                                case configs.onConfirmDisabled of
```

Comment:
Using just a Bool for this type removes the need for this case

Reply:
Yup! I can remove the Maybe. But just confirming.. it will still have the `case configs.onConfirm` as it was already there... and for `configs.onConfirmDisabled` instead of having 2 cases of `Just True and _`, it'll be `True` and `False` ?